src/app.controller.put.ts

```
@@ -147,4 +147,174 @@
+				role_where.push((permission as AuthTablePermissionSuccessResponse).restriction)
+			}
+		}
+
+		//validate :id field
+		const primary_key = this.schema.getPrimaryKey(schema)
+
+		if (!primary_key) {
+			return res.status(400).send(this.response.text(`No primary key found for table ${table_name}`))
+		}
+
+		if (body instanceof Array) {
```

Biome

[error] Use Array.isArray() instead of instanceof Array.

instanceof Array returns false for array-like objects and arrays from other execution contexts.
Unsafe fix: Use Array.isArray() instead.


(lint/suspicious/useIsArray)